.github/workflows/build-test.yml

```diff
@@ -105,3 +105,13 @@ jobs:
 
       - name: Check formatting with Prettier
         run: yarn prettier:check
+
+  clean-up:
+    name: "Clean Up"
+    runs-on: ubuntu-latest
+    needs: [test-sdk, test-ui]
```

Comment:
I would add `if: ${{ always() }}` here, so if one of them fails, we still delete the artifact